msgid ""
msgstr "Project-Id-Version: pyload\n"
"Report-Msgid-Bugs-To: 'bugs@pyload.org'\n"
"POT-Creation-Date: 2014-04-08 15:01+0200\n"
"PO-Revision-Date: 2014-04-09 05:39-0400\n"
"Last-Translator: pyloadTeam <team@pyload.org>\n"
"Language-Team: Galician\n"
"Language: gl_ES\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.net\n"

#: pyload/Core.py:182
msgid "Received Quit signal"
msgstr "Sinal de saída recibida"

#: pyload/Core.py:303
#, python-format
msgid "pyLoad already running with pid %s"
msgstr "pyLoad xa está executándose con pid %s"

#: pyload/Core.py:317
#, python-format
msgid "Failed changing group: %s"
msgstr "Cambio de grupo falido: %s"

#: pyload/Core.py:327
#, python-format
msgid "Failed changing user: %s"
msgstr "Cambio de usuario falido: %s"

#: pyload/Core.py:329
msgid "folder for logs"
msgstr "cartafol para os rexistros"

#: pyload/Core.py:340
msgid "Starting"
msgstr "Comezando"

#: pyload/Core.py:341
#, python-format
msgid "Using home directory: %s"
msgstr "Usando directorio de inicio: %s"

#: pyload/Core.py:350
msgid "pycrypto to decode container files"
msgstr "pycrypto para descodificar contedores de ligazóns"

#: pyload/Core.py:353
msgid "folder for temporary files"
msgstr "cartafol para ficheiros temporais"

#: pyload/Core.py:358
msgid "folder for downloads"
msgstr "cartafol para as descargas"

#: pyload/Core.py:361
msgid "OpenSSL for secure connection"
msgstr "OpenSSL para conexión segura"

#: pyload/Core.py:365
msgid "Moving old user config to DB"
msgstr "Movendo a antiga configuración de usuario á BD"

#: pyload/Core.py:368
msgid "Please check your logindata with ./pyload.py -u"
msgstr "Por favor, comprobe os seus datos de inicio de sesión con ./pyload.py -u"

#: pyload/Core.py:371
msgid "All links removed"
msgstr "Todas as ligazóns foron eliminadas"

#: pyload/Core.py:402
#, python-format
msgid "Downloadtime: %s"
msgstr "Tempo de descarga: %s"

#: pyload/Core.py:412
#, python-format
msgid "Free space: %s"
msgstr "Espazo libre: %s"

#: pyload/Core.py:432
msgid "Activating Accounts..."
msgstr "Activando Contas..."

#: pyload/Core.py:438
msgid "Activating Plugins..."
msgstr "Activando Engadidos..."

#: pyload/Core.py:441
msgid "pyLoad is up and running"
msgstr "pyLoad está funcionando"

#: pyload/Core.py:460
msgid "restarting pyLoad"
msgstr "reiniciando pyLoad"

#: pyload/Core.py:464
msgid "pyLoad quits"
msgstr "pyLoad pechouse"

#: pyload/Core.py:521
#, python-format
msgid "Install %s"
msgstr "Instalar %s"

#: pyload/Core.py:557
#, python-format
msgid "could not find %(desc)s: %(name)s"
msgstr "non se puido atopar %(desc)s: %(name)s"

#: pyload/Core.py:559
#, python-format
msgid "could not create %(desc)s: %(name)s"
msgstr "no se puido crear %(desc)s: %(name)s"

#: pyload/Core.py:580
msgid "shutting down..."
msgstr "apagando..."

#: pyload/Core.py:597
msgid "error while shutting down"
msgstr "produciuse un erro mentres se apagaba"

#: pyload/Core.py:661
msgid "killed pyLoad from Terminal"
msgstr "pyload terminado dende o Terminal"

#: pyload/database/DatabaseBackend.py:174
msgid "Filedatabase was deleted due to incompatible version."
msgstr "O ficheiro da base de datos eliminouse por ser dunha versión incompatible."

#: pyload/database/DatabaseBackend.py:189
msgid "Filedatabase could NOT be converted."
msgstr "O ficheiro da base de datos NON se puido converter."

#: pyload/database/DatabaseBackend.py:198
msgid "Database was converted from v2 to v3."
msgstr "A base de datos converteuse da v2 á v3."

#: pyload/database/DatabaseBackend.py:206
msgid "Database was converted from v3 to v4."
msgstr "A base de datos converteuse da v3 á v4."

#: pyload/database/DatabaseBackend.py:252
msgid "Converting old Django DB"
msgstr "Convertendo a antiga BD Django"

#: pyload/database/FileDatabase.py:45
msgid "finished"
msgstr "finalizado"

#: pyload/database/FileDatabase.py:45
msgid "offline"
msgstr "fora de liña"

#: pyload/database/FileDatabase.py:45
msgid "online"
msgstr "en liña"

#: pyload/database/FileDatabase.py:45
msgid "queued"
msgstr "na cola"

#: pyload/database/FileDatabase.py:45
msgid "skipped"
msgstr "omitido"

#: pyload/database/FileDatabase.py:45
msgid "waiting"
msgstr "esperando"

#: pyload/database/FileDatabase.py:45
msgid "temp. offline"
msgstr "temporalmente fora de servicio"

#: pyload/database/FileDatabase.py:45
msgid "starting"
msgstr "comezando"

#: pyload/database/FileDatabase.py:45
msgid "failed"
msgstr "falido"

#: pyload/database/FileDatabase.py:45
msgid "aborted"
msgstr "abortado"

#: pyload/database/FileDatabase.py:45
msgid "decrypting"
msgstr "descifrando"

#: pyload/database/FileDatabase.py:45
msgid "custom"
msgstr "personalizado"

#: pyload/database/FileDatabase.py:45
msgid "downloading"
msgstr "descargando"

#: pyload/database/FileDatabase.py:45
msgid "processing"
msgstr "procesando"

#: pyload/database/FileDatabase.py:45
msgid "unknown"
msgstr "descoñecido"

#: pyload/database/FileDatabase.py:531 pyload/plugins/addon/IRCInterface.py:74
#: pyload/plugins/addon/XMPPInterface.py:83
#, python-format
msgid "Package finished: %s"
msgstr "Paquete finalizado: %s"

#: pyload/remote/ThriftBackend.py:39
msgid "Using SSL ThriftBackend"
msgstr "Usando SSL ThriftBackend"

#: pyload/manager/RemoteManager.py:35
#, python-format
msgid "Remote backend error: %s"
msgstr "Erro remoto do motor: %s"

#: pyload/manager/RemoteManager.py:82
#, python-format
msgid "Starting %(name)s: %(addr)s:%(port)s"
msgstr "Iniciando %(name)s: %(addr)s:%(port)s"

#: pyload/manager/RemoteManager.py:84
#, python-format
msgid "Failed loading backend %(name)s | %(error)s"
msgstr "Fallou ó cargar o motor %(name)s | %(error)s"

#: pyload/web/json_app.py:57
#, python-format
msgid "waiting %s"
msgstr "esperando %s"

#: pyload/thread/ServerThread.py:35
msgid "SSL certificates not found."
msgstr "Certificados SSL non encontrados."

#: pyload/thread/ServerThread.py:39
#, python-format
msgid "Sorry, we dropped support for starting %s directly within pyLoad"
msgstr "Sentímolo, pero retirámolo soporte para iniciar %s directamente en pyLoad"

#: pyload/thread/ServerThread.py:40
msgid "You can use the threaded server which offers good performance and ssl,"
msgstr "Pode empregar o servidor con fíos que ofrece un bo rendemento e ssl,"

#: pyload/thread/ServerThread.py:41
#, python-format
msgid "of course you can still use your existing %s with pyLoads fastcgi server"
msgstr "por suposto pode seguir empregando o seu %s existente co servidor fastcgi de pyLoad"

#: pyload/thread/ServerThread.py:42
msgid "sample configs are located in the pyload/web/servers directory"
msgstr "as configuracións de exemplo encóntranse no directorio pyload/web/servers"

#: pyload/thread/ServerThread.py:49
#, python-format
msgid "Can't use %(server)s, python-flup is not installed!"
msgstr "Non pode usar %(server)s, python-flup non está instalado!"

#: pyload/thread/ServerThread.py:56
#, python-format
msgid "Error importing lightweight server: %s"
msgstr "Erro importando servidor lixeiro: %s"

#: pyload/thread/ServerThread.py:57
msgid "You need to download and compile bjoern, https://github.com/jonashaag/bjoern"
msgstr "Debe descargar e compilar bjoern, https://github.com/jonashaag/bjoern"

#: pyload/thread/ServerThread.py:58
msgid "Copy the boern.so to pyload/lib folder or use Setup.py install"
msgstr "Copie o módulo boern.so ó cartafol pyload/lib ou use a instalación Setup.py"

#: pyload/thread/ServerThread.py:59
msgid "Of course you need to be familiar with linux and know how to compile software"
msgstr "Evidentemente, debe estar familiarizado con linux e saber como compilar software"

#: pyload/thread/ServerThread.py:63
msgid "Server set to threaded, due to known performance problems on windows."
msgstr "Servidor en modo threaded, debido a problemas de rendemento coñecidos en windows."

#: pyload/thread/ServerThread.py:80 pyload/thread/ServerThread.py:103
msgid "This server offers no SSL, please consider using threaded instead"
msgstr "Este servidor non ofrece SSL, por favor considere usar o servidor con fíos no seu lugar"

#: pyload/thread/ServerThread.py:82
#, python-format
msgid "Starting builtin webserver: %(host)s:%(port)d"
msgstr "Iniciando servidor web incorporado: %(host)s:%(port)d"

#: pyload/thread/ServerThread.py:87
#, python-format
msgid "Starting threaded SSL webserver: %(host)s:%(port)d"
msgstr "Iniciando servidor web SSL con fíos: %(host)s:%(port)d"

#: pyload/thread/ServerThread.py:91
#, python-format
msgid "Starting threaded webserver: %(host)s:%(port)d"
msgstr "Iniciando servidor web con fíos: %(host)s:%(port)d"

#: pyload/thread/ServerThread.py:97
#, python-format
msgid "Starting fastcgi server: %(host)s:%(port)d"
msgstr "Iniciando servidor fastcgi: %(host)s:%(port)d"

#: pyload/thread/ServerThread.py:105
#, python-format
msgid "Starting lightweight webserver (bjoern): %(host)s:%(port)d"
msgstr "Iniciando servidor web lixeiro (bjoern): %(host)s:%(port)d"

#: pyload/web/pyload_app.py:125
msgid "You dont have permission to access this page."
msgstr "Non ten permisos para acceder a esta páxina."

#: pyload/web/pyload_app.py:193
msgid "Download directory not found."
msgstr "Non se encontra o directorio de descarga."

#: pyload/web/pyload_app.py:260 pyload/web/pyload_app.py:267
msgid "unlimited"
msgstr ""

#: pyload/web/pyload_app.py:262 pyload/web/pyload_app.py:269
msgid "not available"
msgstr "non dispoñible"

#: pyload/web/pyload_app.py:508
msgid "Run pyload.py -s to access the setup."
msgstr "Execute pyload.py -s para acceder á instalación."

#: pyload/network/HTTPDownload.py:245
#, python-format
msgid "Download chunks failed, fallback to single connection | %s"
msgstr "Descarga por partes falida, conexión única de emerxencia| %s"

#: pyload/thread/PluginThread.py:183
#, python-format
msgid "Download starts: %s"
msgstr "Iniciando descarga: %s"

#: pyload/thread/PluginThread.py:189
#, python-format
msgid "Download finished: %s"
msgstr "Descarga finalizada: %s"

#: pyload/thread/PluginThread.py:194 pyload/thread/PluginThread.py:365
#, python-format
msgid "Plugin %s is missing a function."
msgstr "O engadido %s non atopa ningunha función."

#: pyload/thread/PluginThread.py:202 pyload/thread/PluginThread.py:265
#: pyload/thread/PluginThread.py:382
#, python-format
msgid "Download aborted: %s"
msgstr "Descarga cancelada: %s"

#: pyload/thread/PluginThread.py:222
#, python-format
msgid "Download restarted: %(name)s | %(msg)s"
msgstr "Descarga reiniciada: %(name)s | %(msg)s"

#: pyload/thread/PluginThread.py:231 pyload/thread/PluginThread.py:373
#, python-format
msgid "Download is offline: %s"
msgstr "A descarga está fóra de servizo: %s"

#: pyload/thread/PluginThread.py:234
#, python-format
msgid "Download is temporary offline: %s"
msgstr "A descarga está temporalmente fora de servizo: %s"

#: pyload/thread/PluginThread.py:237 pyload/thread/PluginThread.py:304
#, python-format
msgid "Download failed: %(name)s | %(msg)s"
msgstr "Descarga falida: %(name)s | %(msg)s"

#: pyload/thread/PluginThread.py:254
msgid "Couldn't connect to host or connection reset, waiting 1 minute and retry."
msgstr "No se puido conectar ó servidor ou conexión reiniciada, esperando 1 minuto e tentando de novo."

#: pyload/thread/PluginThread.py:290
#, python-format
msgid "Download skipped: %(name)s due to %(plugin)s"
msgstr "Descarga omitida: %(name)s debido a %(plugin)s"

#: pyload/thread/PluginThread.py:361
#, python-format
msgid "Decrypting starts: %s"
msgstr "Comeza o descifrado: %s"

#: pyload/thread/PluginThread.py:376 pyload/thread/PluginThread.py:394
#, python-format
msgid "Decrypting failed: %(name)s | %(msg)s"
msgstr "Descifrado falido: %(name)s | %(msg)s"

#: pyload/thread/PluginThread.py:388
#, python-format
msgid "Retrying %s"
msgstr "Reintentando %s"

#: pyload/thread/PluginThread.py:635
#, python-format
msgid "Info Fetching for %(name)s failed | %(err)s"
msgstr "A Obtención de información para %(name)s fallou | %(err)s"

#: pyload/manager/AddonManager.py:90 pyload/plugins/Addon.py:102
#, python-format
msgid "Error executing addon: %s"
msgstr "Erro executando as ligazóns: %s"

#: pyload/manager/AddonManager.py:140
#, python-format
msgid "Failed activating %(name)s"
msgstr "Fallou a activación de %(name)s"

#: pyload/manager/AddonManager.py:144
#, python-format
msgid "Activated plugins: %s"
msgstr "Engadidos activados: %s"

#: pyload/manager/AddonManager.py:145
#, python-format
msgid "Deactivate plugins: %s"
msgstr "Engadidos desactivados: %s"

#: pyload/manager/ThreadManager.py:137
#, python-format
msgid "Reconnect Failed: %s"
msgstr "Reconexión Falida: %s"

#: pyload/manager/ThreadManager.py:176
msgid "Reconnect script not found!"
msgstr "Non se atopou o script de reconexión!"

#: pyload/manager/ThreadManager.py:182
msgid "Starting reconnect"
msgstr "Iniciando reconexión"

#: pyload/manager/ThreadManager.py:196
msgid "Failed executing reconnect script!"
msgstr "Fallo executando o script de reconexión!"

#: pyload/manager/ThreadManager.py:208
#, python-format
msgid "Reconnected, new IP: %s"
msgstr "Conectado de novo, nova dirección IP: %s"

#: pyload/manager/ThreadManager.py:288
msgid "Not enough space left on device"
msgstr "Non hai suficiente espazo libre no dispositivo"

#: pyload/plugins/Account.py:85 pyload/plugins/Account.py:91
#, python-format
msgid "Could not login with account %(user)s | %(msg)s"
msgstr "Non se puido iniciar sesión coa conta %(user)s | %(msg)s"

#: pyload/plugins/Account.py:86
msgid "Wrong Password"
msgstr "Contrasinal erróneo"

#: pyload/plugins/Account.py:240
#, python-format
msgid "Your Time %s has wrong format, use: 1:22-3:44"
msgstr "A hora %s ten un formato incorrecto, use: 1:22-3:44"

#: pyload/plugins/Account.py:266
#, python-format
msgid "Account %s has not enough traffic, checking again in 30min"
msgstr "A conta %s non ten tráfico suficiente, comprobarase outra vez en 30min"

#: pyload/plugins/Account.py:273
#, python-format
msgid "Account %s is expired, checking again in 1h"
msgstr "A conta %s expirou, compróbea de novo en 1h"

#: pyload/plugins/crypter/SerienjunkiesOrg.py:128
msgid "Downloadlimit reached"
msgstr "Limite de descargas alcanzado"

#: pyload/manager/PluginManager.py:153
#, python-format
msgid "%s has a invalid pattern."
msgstr "%s ten un patrón inválido."

#: pyload/manager/PluginManager.py:272
#, python-format
msgid "Error importing %(name)s: %(msg)s"
msgstr "Erro importando %(name)s: %(msg)s"

#: pyload/plugins/internal/MultiHoster.py:133
msgid "No Hoster loaded"
msgstr "Non se cargou ningún servidor"

#: pyload/plugins/account/BitshareCom.py:37
msgid "Activate direct Download in your Bitshare Account"
msgstr "Active a descarga directa na súa conta de Bitshare"

#: pyload/plugins/container/LinkList.py:65
msgid "LinkList could not be cleared."
msgstr "A lista de ligazóns non se puido limpar."

#: pyload/manager/AccountManager.py:88
msgid "Account settings deleted, due to new config format."
msgstr "Configuración da conta borrada, debido ó formato da nova configuración."

#: pyload/plugins/hoster/BasePlugin.py:65
msgid "Authorization required (username:password)"
msgstr "Autorización requirida (usuario:contrasinal)"

#: pyload/plugins/hoster/SimplydebridCom.py:25
#: pyload/plugins/hoster/RealdebridCom.py:41
#: pyload/plugins/hoster/FreeWayMe.py:38 pyload/plugins/hoster/ZeveraCom.py:22
#: pyload/plugins/hoster/UnrestrictLi.py:53
#: pyload/plugins/hoster/Premium4Me.py:28 pyload/plugins/hoster/FastixRu.py:36
#: pyload/plugins/hoster/AlldebridCom.py:38
#: pyload/plugins/hoster/DebridItaliaCom.py:40
#: pyload/plugins/hoster/RPNetBiz.py:26
#: pyload/plugins/hoster/MultiDebridCom.py:41
#: pyload/plugins/hoster/ReloadCc.py:24 pyload/plugins/hoster/RehostTo.py:26
#: pyload/plugins/hoster/PremiumizeMe.py:22
#: pyload/plugins/addon/RPNetBiz.py:43
#, python-format
msgid "Please enter your %s account or deactivate this plugin"
msgstr ""

#: pyload/plugins/hoster/FilesMailRu.py:99
#, python-format
msgid "There was HTML Code in the Downloaded File (%s)...redirect error? The Download will be restarted."
msgstr ""

#: pyload/plugins/hoster/NetloadIn.py:146
#: pyload/plugins/hoster/NetloadIn.py:170
msgid "File temporarily not available"
msgstr "Arquivo non dispoñible temporalmente"

#: pyload/plugins/hoster/NetloadIn.py:183
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr "Netload: esperando entre descargas %d s."

#: pyload/plugins/hoster/NetloadIn.py:214
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr "Netload: esperando captcha %d s."

#: pyload/plugins/hoster/NetloadIn.py:252
msgid "Downloaded File was empty"
msgstr "O arquivo descargado estaba baleiro"

#: pyload/plugins/hoster/UploadedTo.py:129
msgid "API key invalid"
msgstr "Clave de API inválida"

#: pyload/plugins/hoster/UploadedTo.py:153
#, python-format
msgid "%s: Not enough traffic left"
msgstr "%s: Non queda suficiente tráfico"

#: pyload/plugins/hoster/UploadedTo.py:156
msgid "Traffic exceeded"
msgstr "Tráfico excedido"

#: pyload/plugins/hoster/RapidshareCom.py:100
msgid "Rapidshare: Traffic Share (direct download)"
msgstr "Rapidshare: Tráfico compartido (descarga directa)"

#: pyload/plugins/hoster/RapidshareCom.py:127
#: pyload/plugins/hoster/RapidshareCom.py:194
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr "Xa está descargando dende esta dirección ip, esperando 60 segundos"

#: pyload/plugins/hoster/RapidshareCom.py:131
msgid "Invalid Auth Code, download will be restarted"
msgstr "Código de Autenticación Inválido, a descarga reiniciarase"

#: pyload/plugins/hoster/RapidshareCom.py:199
msgid "RapidShareCom: No free slots"
msgstr "RapidShareCom: No quedan espazos dispoñibles"

#: pyload/plugins/hoster/RapidshareCom.py:202
msgid "You need a premium account for this file"
msgstr "Precisa dunha conta premium para este arquivo"

#: pyload/plugins/hoster/RapidshareCom.py:204
msgid "Filename reported invalid"
msgstr "Nome do arquivo inválido"

#: pyload/plugins/hoster/FileserveCom.py:99
msgid "Parallel download error, now waiting 60s."
msgstr "Erro con descargas en paralelo, esperando 60s."

#: pyload/plugins/hoster/FileserveCom.py:215
msgid "Not logged in."
msgstr "Non iniciou sesión."

#: pyload/plugins/hoster/MegaNz.py:56
msgid "Decryption failed"
msgstr ""

#: pyload/plugins/hoster/MegaNz.py:106
msgid "No file key provided in the URL"
msgstr ""

#: pyload/plugins/hoster/MegaNz.py:118
msgid "Error code:"
msgstr ""

#: pyload/plugins/Container.py:68
msgid "File not exists."
msgstr "O ficheiro non existe."

#: pyload/plugins/addon/UpdateManager.py:74
msgid "*** Plugins have been updated, please restart pyLoad ***"
msgstr "*** Os engadidos foron actualizados, por favor reinicie pyLoad ***"

#: pyload/plugins/addon/UpdateManager.py:76
msgid "Plugins updated and reloaded"
msgstr "Engadidos actualizados e cargados de novo"

#: pyload/plugins/addon/UpdateManager.py:79
msgid "No plugin updates available"
msgstr "Non hai actualizacións de engadidos dispoñibles"

#: pyload/plugins/addon/UpdateManager.py:96
msgid "No Updates for pyLoad"
msgstr "No hai actualizacións para pyLoad"

#: pyload/plugins/addon/UpdateManager.py:100
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr "***  Nova versión %s de pyLoad dispoñible  ***"

#: pyload/plugins/addon/UpdateManager.py:101
msgid "***  Get it here: http://pyload.org/download  ***"
msgstr "***  Obtéñaa aquí: http://pyload.org/download  ***"

#: pyload/plugins/addon/UpdateManager.py:104
msgid "Not able to connect server for updates"
msgstr "Imposible conectar co servidor para actualizacións"

#: pyload/plugins/addon/UpdateManager.py:148
#, python-format
msgid "New version of %(type)s|%(name)s : %(version).2f"
msgstr "Nova versión de %(type)s|%(name)s : %(version).2f"

#: pyload/plugins/addon/UpdateManager.py:157
#: pyload/plugins/addon/UpdateManager.py:162
#, python-format
msgid "Error when updating %s"
msgstr "Erro mentres se actualizaba %s"

#: pyload/plugins/addon/UpdateManager.py:162
msgid "Version mismatch"
msgstr "Non coincide a versión"

#: pyload/plugins/addon/IRCInterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr "Descarga finalizada: %(name)s @ %(plugin)s "

#: pyload/plugins/addon/IRCInterface.py:95
#, python-format
msgid "New Captcha Request: %s"
msgstr "Nova Petición de Captcha: %s"

#: pyload/plugins/addon/IRCInterface.py:96
#, python-format
msgid "Answer with 'c %s text on the captcha'"
msgstr "Responda con 'c %s texto ó captcha'"

#: pyload/plugins/addon/Premium4Me.py:29
msgid "Please add your premium.to account first and restart pyLoad"
msgstr ""

#: pyload/plugins/addon/HotFolder.py:82
#, python-format
msgid "Added %s from HotFolder"
msgstr "Engadido %s dende HotFolder"

#: pyload/plugins/addon/ExtractArchive.py:95
#, python-format
msgid "No %s installed"
msgstr "%s non instalado"

#: pyload/plugins/addon/ExtractArchive.py:97
#: pyload/plugins/addon/ExtractArchive.py:102
#, python-format
msgid "Could not activate %s"
msgstr "Non se puido activar %s"

#: pyload/plugins/addon/ExtractArchive.py:107
msgid "Activated"
msgstr "Activado"

#: pyload/plugins/addon/ExtractArchive.py:109
msgid "No Extract plugins activated"
msgstr "Non se activaron os engadidos de extracción"

#: pyload/plugins/addon/ExtractArchive.py:121
#, python-format
msgid "Package %s queued for later extracting"
msgstr "Paquete %s posto na cola para a súa extracción posterior"

#: pyload/plugins/addon/ExtractArchive.py:144
#, python-format
msgid "Check package %s"
msgstr "Comprobar paquete %s"

#: pyload/plugins/addon/ExtractArchive.py:185
#, python-format
msgid "Extract to %s"
msgstr "Extraer a %s"

#: pyload/plugins/addon/ExtractArchive.py:200
msgid "No files found to extract"
msgstr ""

#: pyload/plugins/addon/ExtractArchive.py:207
msgid "extracting"
msgstr "extraendo"

#: pyload/plugins/addon/ExtractArchive.py:218
msgid "Password protected"
msgstr "Protexido con contrasinal"

#: pyload/plugins/addon/ExtractArchive.py:239
msgid "Wrong password"
msgstr "Contrasinal erróneo"

#: pyload/plugins/addon/ExtractArchive.py:247
#, python-format
msgid "Deleting %s files"
msgstr "Borrando ficheiros %s"

#: pyload/plugins/addon/ExtractArchive.py:254
msgid "Extracting finished"
msgstr "Extracción finalizada"

#: pyload/plugins/addon/ExtractArchive.py:260
msgid "Archive Error"
msgstr "Erro de Arquivo"

#: pyload/plugins/addon/ExtractArchive.py:262
msgid "CRC Mismatch"
msgstr "O CRC non coincide"

#: pyload/plugins/addon/ExtractArchive.py:266
msgid "Unknown Error"
msgstr "Erro Descoñecido"

#: pyload/plugins/addon/ExtractArchive.py:318
msgid "Setting User and Group failed"
msgstr "Configuración de Usuario e Grupo falida"

#: pyload/plugins/addon/ClickAndLoad.py:75
msgid "Click'N'Load: Port 9666 already in use"
msgstr "Click'N'Load: O porto 9666 xa está en uso"

#: pyload/plugins/addon/CaptchaTrader.py:69
#: pyload/plugins/addon/Captcha9kw.py:59
#: pyload/plugins/addon/ExpertDecoders.py:50
#, python-format
msgid "%s credits left"
msgstr "%s créditos restantes"

#: pyload/plugins/addon/CaptchaTrader.py:117
msgid "Could not send response."
msgstr ""

#: pyload/plugins/addon/CaptchaTrader.py:135
msgid "Your CaptchaTrader Account has not enough credits"
msgstr "A súa conta de CaptchaTrader non ten créditos suficientes"

#: pyload/plugins/addon/LinkdecrypterCom.py:43
msgid "Crypter list not found"
msgstr ""

#: pyload/plugins/addon/LinkdecrypterCom.py:57
msgid "Crypter list is empty"
msgstr ""

#: pyload/plugins/addon/XMPPInterface.py:91
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr "Descarga finalizada: %(name)s @ %(plugin)s"

#: pyload/plugins/addon/Captcha9kw.py:93
#, python-format
msgid "New CaptchaID from upload: %s : %s"
msgstr ""

#: pyload/plugins/addon/Captcha9kw.py:129
msgid "Your Captcha 9kw.eu Account has not enough credits"
msgstr ""

#: pyload/plugins/addon/ExternalScripts.py:54
#, python-format
msgid "Installed scripts for %s: "
msgstr "Scripts instalados para %s: "

#: pyload/plugins/addon/ExternalScripts.py:69
msgid "Script not executable:"
msgstr "Script non executable:"

#: pyload/plugins/addon/ExternalScripts.py:80
#, python-format
msgid "Error in %(script)s: %(error)s"
msgstr "Erro en %(script)s: %(error)s"

#: pyload/plugins/addon/ExpertDecoders.py:95
msgid "Your ExpertDecoders Account has not enough credits"
msgstr ""

#: pyload/plugins/addon/RehostTo.py:32
msgid "Please add your rehost.to account first and restart pyLoad"
msgstr "Por favor, engada primeiro a súa conta rehost.to e reinicie pyLoad"

#: pyload/plugins/addon/PremiumizeMe.py:48
msgid "Please add a valid premiumize.me account first and restart pyLoad."
msgstr ""

#: pyload/plugins/addon/CaptchaBrotherhood.py:69
#, python-format
msgid "%d credits left"
msgstr ""

#: pyload/plugins/Base.py:389
msgid "Pil and tesseract not installed and no Client connected for captcha decrypting"
msgstr "Pil e tesseract non están instalados e no hai cliente conectado para descifrar o captcha"

#: pyload/plugins/Base.py:393
msgid "No captcha result obtained in appropiate time by any of the plugins."
msgstr "Non se obtiveron resultados para o captcha no tempo asignado de ningún dos engadidos."

#: pyload/plugins/Base.py:498 pyload/plugins/Base.py:532
#, python-format
msgid "Setting User and Group failed: %s"
msgstr "Configuración de Usuario e Grupo falida: %s"

#: pyload/manager/CaptchaManager.py:78
msgid "No Client connected for captcha decrypting"
msgstr "Non hai cliente conectado para descifrar o captcha"

#: pyload/Api.py:330
#, python-format
msgid "Added package %(name)s containing %(count)d links"
msgstr "Engadido o paquete %(name)s que contén %(count)d ligazóns"

#: pyload/Api.py:593
#, python-format
msgid "Added %(count)d links to package #%(package)d "
msgstr "Engadidos %(count)d ligazóns ó paquete #%(package)d "

#: pyload/utils/JsEngine.py:156
msgid "No js engine detected, please install either Spidermonkey, ossp-js, pyv8 or rhino"
msgstr "Non se detectou ningún motor de js, por favor instale Spidermonkey, ossp-js, pyv8 ou rhino"

